cli/cmd/get_project.go

```diff
@@ -0,0 +1,155 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/keptn/go-utils/pkg/api/models"
+	apiutils "github.com/keptn/go-utils/pkg/api/utils"
+	keptn "github.com/keptn/go-utils/pkg/lib"
+	"github.com/keptn/keptn/cli/pkg/credentialmanager"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+type getProjectStruct struct {
+	outputFormat *string
+}
+
+var getProject getProjectStruct
+
+var getProjectCmd = &cobra.Command{
+	Use:   "project",
```

Comment:
according to the Issue, the better approach would be to merge the command definitions for the singular and the plural into one command, using the `Aliases` property:

```
var getProjectCmd = &cobra.Command{
	Use:   "project",
        Aliases: []string{"projects"},
	Short: "Get details of a keptn project",
	Long:  `Get details of a given keptn project`,
	Example: `keptn get project sockshop
// ...
```

The same holds for the `get stage(s)` and `get service(s)` commands

Reply:
Ok, will look into it and try to merge these two commands into a single file, but not sure how to handle required arguments like --project and the different help text on the different sub-commands. 

If you have an example how I can do that would be good.

Reply:
Thank you @bacherfl, I've merged the commands.